src/lib/locales/en.json

```diff
@@ -97,6 +97,10 @@
 		"qasim": {
 			"catchphrase": "☁boy 🤠",
 			"body": "Software Engineer. Excited by cutting edge technologies and languages. Always on the hunt for the sharpest knife in the drawer."
+		},
+		"philippe": {
+			"catchphrase": "Software Wizzard",
```

Comment:
`Wizard` is spelled with one `z`. Is the word `Wizzard` intentional?

Reply:
See above